dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
I guess another advantage is that the postgresql database, redis, memcache, and elasticsearch also run in containers, and not on the host.

This means that I can just run `docker-compose down` to stop everything when I'm done. On the host I'll have to remember to shut down the services.